check_os.py

```diff
@@ -11,8 +12,9 @@
 from Tribler.Core.Config.tribler_config import TriblerConfig
 from Tribler.Core.Modules.process_checker import ProcessChecker
 
-FORCE_RESTART_MESSAGE = "A Tribler instance is already running. Do you want to force restart? " \
-                        "\n\nCaution: force restart could result in data corruption."
+FORCE_RESTART_MESSAGE = "An existing Tribler core instance (PID:%s) is already running. \n\n" \
+                        "Do you want to do a clean restart (YES) or use the existing core (NO)?"
```

Comment:
Maybe leave the second part (use the existing core...) out, I think this message is a bit confusing otherwise.